README.md

```diff
@@ -146,3 +146,32 @@ apl
 
 * tplgtool.py
 <br> Dumps info from tplg binary file.
+
+
+## System configuration tips
+
+### sudo noise
+
+sof-test uses sudo a lot which creates a lot of noise in `journalctl`
+output which is especially a problem when testing. To turn off that
+noise first run `sudo visudo` and add the following line:
+
+```
+Defaults:your_unix_id,root !log_allowed
+```
+For more see `man sudoers`.
+
+Then add the following line at the _top_ of `/etc/pam.d/sudo`
+
+```
+# Be "done" when the pam_succeed_if.so arguments are matched; don't
+# process other lines. If not matched then "ignore" this line.
+# Also support "double sudo" :-(
+session [success=done default=ignore] pam_succeed_if.so quiet         uid = 0 ruser = root
+session [success=done default=ignore] pam_succeed_if.so quiet_success uid = 0 ruser = your_id
```

Comment:
@marc-hb @fredoh9 is this enabled in deploy scripts?

Reply:
No, I don't think so